### ISSUE TITLE
Error installing application sources.

### ISSUE THREAD

**Issue by kmcgowan0** [alt3/cakebox#94]
When running ` cakebox application add mycake3.app` , I get the error `Error installing application sources.`

I've checked the error log, and it seems to be a permissions problem:

` [ErrorException] copy(/home/vagrant/.composer/cache/files/squizlabs/php_codesniffer/21fc9024bccaa50ebc7c3aa8aebb590119cbfa32.zip): failed to open stream: Permission denied","@tags":["cli.cakebox"],"@type":"cakephp"}`

I've tried increasing the memory on the box, but no luck.

Any help would be greatly appreciated.

Thanks!

**Comment by bravo-kernel** [comment]
Yes, unfortunately this slipped in somewhere. The workaround until patched as copied from https://github.com/alt3/cakebox/issues/91#issuecomment-364211673:

# FIX FOR PERMISSION ERROR WHEN CREATING A NEW APPLICATION 

I got a permission error creating a new application. Fix by running:

```bash
sudo chown vagrant:vagrant /home/vagrant/.composer/cache/files/squizlabs -R
```  

Please close ticket if this resolves the issue for you.

**Comment by bravo-kernel** [comment]
Totally agree and I think the fix is as easy as adding the command to https://github.com/alt3/cakebox-console/blob/dev/src/Shell/UpdateShell.php probably.
```
sudo chown vagrant:vagrant /home/vagrant/.composer/cache/files/squizlabs -R
```

**Comment by bravo-kernel** [comment]
All box fixes should be done there. If you can wait a week then I will add it, if not, feel free to add it yourself.

**Comment by dereuromark** [comment]
The whole cache :) not just this one repo.
But  I guess we can wait on your fix.

**Comment by bravo-kernel** [comment]
ok, fair enough

**Comment by bravo-kernel** [comment]
Fixed through #100 💯
